### PR TITLE
Update aiidalab, aiidalab-qe, temporarily remove aiida-siesta plugin.

### PR DESCRIPTION
**Pull request by yakutovicha**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ aiida-optimize~=0.4
 aiida-qeq~=1.0.0a1
 aiida-quantumespresso~=3.4
 aiida-raspa~=1.2
-aiida-siesta~=1.1
+#aiida-siesta~=1.1
```
Comment (csadorf) [inline requirements.txt:18]: If we comment it out, then we should also provide a reason, otherwise just remove it.

Comment (yakutovicha) [reply requirements.txt]: removed.

**Review by csadorf — APPROVED** [review]
Looks fine to me, however I did not test it.

**Comment by yakutovicha** [comment]
> Looks fine to me, however I did not test it.

ok, thanks. The image will remain in the dev instance for testing for one week.

**Pull request merged**